### PR TITLE
Language updates

### PR DESCRIPTION
**Pull request by thekaveman**
## Finished

* [x] Update English text based on recent feedback
* [x] Refactor code to mark all translatable (user-facing) strings
* [x] Introduce PO files for English and Spanish text
* [x] Implement language per-request detection and toggling via menu button

## Outstanding

* [ ] Update Spanish text with new translations, including missing translations
* [ ] Add `django-admin compilemessages` to container startup sequence

## Notes

This branch was made on top of the work in #9, which implements the remaining app screens. 

This PR should not be merged until after #9.

### CODE REVIEW

File: server/static/tokenize.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+tokenize = function(args) {
+    console.log(args);
```
Comment (krainboltgreene) [inline server/static/tokenize.js:2]: TODO: Make sure this gets changed

**Review by krainboltgreene — APPROVED** [review]
Really loving this solution!

File: server/static/tokenize.js (hunk 1 above)
Comment (thekaveman) [reply server/static/tokenize.js:2]: This is our test verification server, will not be part of production deployment.

**Comment by thekaveman** [comment]
Rebased to remove the now-merged commits from #9.

**Comment by thekaveman** [comment]
Getting this into `main` and we'll open additional PRs with translation updates.

**Pull request merged**